### PR TITLE
Expose border crossings, rest stops and toll collection

### PR DESCRIPTION
**Pull request by LukasPaczos**
This PR exposes:
- `leg.admins` (An array of objects describing the administrative boundaries the route leg travels through)
- `intersections.admin_index` (The zero-based index into the admin list on the route leg for this intersection)
- `intersections.rest_stop` (An object containing information about passing rest stops along the route)
- `intersections.toll_collection` (An object containing information about a toll collection point along the route)

### CODE REVIEW

**Review by korshaknn — APPROVED** [review]
LGTM

File: services-directions-models/src/test/java/com/mapbox/api/directions/v5/models/StepIntersectionTest.java [hunk 1]
```diff
@@ -98,4 +101,46 @@ public void testFromJson() {
...
+  @Test
+  public void testAdminIndex() {
+    String stepIntersectionJsonString = "{"
+    + "\"location\": [ 13.426579, 52.508068 ],"
+    + "\"admin_index\": 2"
+    + "}";
+
+    StepIntersection stepIntersection = StepIntersection.fromJson(stepIntersectionJsonString);
+
+    Assert.assertEquals(2, stepIntersection.adminIndex().intValue());
```
Comment (Guardiola31337) [inline services-directions-models/src/test/java/com/mapbox/api/directions/v5/models/StepIntersectionTest.java:114]: `assertEquals(2, stepIntersection.adminIndex().intValue());`

**Review by Guardiola31337 — COMMENTED** [review]
Some minor post-commit comments @LukasPaczos

**Pull request merged**